cobbler/tftpgen.py

```diff
@@ -935,6 +935,48 @@ def build_kernel(
             autoinstall_meta, kernel_path, initrd_path, boot_loader
         )
 
+    def get_serial_kernel_options(self, append_line: str, arch: Archs, system = None):
+        """
+        Builds the console= kernel command line arguement
+
+        If system is none or serial options are not set, defaults are applied:
+        driver -> arch dependent
```

Comment:
That's nonsense. Serial drivers do not depend on architecture but rather machine model.